source/fundamentals/builders/sort.txt

```diff
@@ -32,35 +32,23 @@ To learn more, see our :doc:`guide on builders </fundamentals/builders/>`.
 You should read this guide if you would like to:
 
 * Use builders to specify sort criteria for your queries.
-* Perform ascending sorts and descending sorts.
-* Combine sort criteria.
-* Sort on the text score of a
-  :manual:`text search </core/text-search-operators/>`.
 
 If you want to learn the fundamentals of sorting in the MongoDB Java
 driver, consider reading our
 :doc:`guide on sorting </fundamentals/crud/read-operations/sort/>`.
 
-The Sorts Class
----------------
-
-The :java-docs:`Sorts </apidocs/mongodb-driver-core/com/mongodb/client/model/Sorts.html>`
-class is a builder that provides static factory methods for all sort criteria
-operators supported by MongoDB. These methods return a
-:java-docs:`Bson </apidocs/bson/org/bson/conversions/Bson.html>`
-object that you can pass to the 
-:java-docs:`sort() </apidocs/mongodb-driver-sync/com/mongodb/client/FindIterable.html#sort(org.bson.conversions.Bson)>`
-method of a ``FindIterable`` instance or to 
-:java-docs:`Aggregates.sort() </apidocs/mongodb-driver-core/com/mongodb/client/model/Aggregates.html#sort(org.bson.conversions.Bson)>`.
-If you want to learn more about ``Aggregates``, see our 
-:doc:`guide on the Aggregates builder </fundamentals/builders/aggregates>`.
+.. warning::
 
+   MongoDB does not guarantee sort order in the event of **ties**. A
+   tie occurs when two or more documents contain identical values in the
+   field being used to order your results. For a detailed discussion of ties, 
```

Comment:
Thanks for the note. I added the warning because tied sort behavior seemed to me the main thing about sorting that could trip up a developer in their app, and they have skimmed the intro and not be aware of the other article. 

This probably is assuming too much however, and focusing this guide around the builder and less around sorting is probably for the best. I will remove the warning. Fix: